es-app/src/guis/GuiMenu.cpp

```diff
@@ -4819,7 +4831,38 @@ std::shared_ptr<OptionListComponent<std::string>> GuiMenu::createRatioOptionList
 	return ratio_choice;
 }
 
-std::shared_ptr<OptionListComponent<std::string>> GuiMenu::createVideoResolutionModeOptionList(Window *window, std::string configname) 
+#ifdef _ENABLEEMUELEC
+std::shared_ptr<OptionListComponent<std::string>> GuiMenu::createNativeVideoResolutionModeOptionList(Window *window, std::string configname)
+{
+	auto emuelec_video_mode = std::make_shared< OptionListComponent<std::string> >(window, "NATIVE VIDEO", false);
+			std::vector<std::string> videomode;
+	videomode.push_back("1080p60hz");
+	videomode.push_back("1080i60hz");
+	videomode.push_back("720p60hz");
+	videomode.push_back("720p50hz");
+	videomode.push_back("480p60hz");
+	videomode.push_back("480cvbs");
+	videomode.push_back("576p50hz");
+	videomode.push_back("1080p50hz");
+	videomode.push_back("1080i50hz");
+	videomode.push_back("576cvbs");
```

Comment:
I am not 100% sure we should include the cvbs options, as it requires the AV cable to be connected, and whoever is using CVBS is already on 480 or 576 resolution. I just think it could make it complicated for some people if chosen by mistake, thoughts?

Reply:
Up to you my man! I dont use legacy display so it does not affect me either way.